.github/workflows/ci.yml

```diff
@@ -401,6 +401,8 @@ jobs:
         os: [windows-latest]
     steps:
     - uses: actions/checkout@v2
+    - name: Install mingw
+      run: choco install mingw
```

Comment:
You don't need this. The CI is fine, it was something wrong on someone else's end, not us. (Sorry for the double review)

Reply:
No worries I will remove it